conans/test/integration/run_envronment_test.py

```diff
@@ -85,17 +90,53 @@ def package(self):
                      "main.cpp": main,
                      "hello.cpp": hello_cpp,
                      "hello.h": hello_h})
-        client.run("create . Pkg/0.1@user/testing")
+        client.run("create . Pkg/0.1@lasote/testing")
+        client.run("upload Pkg* --all --confirm")
```

Comment:
Delete all the client cache to make sure it is not pointing to the old cache directory.